src/main/java/core/basesyntax/Pair.java

```diff
@@ -1,5 +1,46 @@
 package core.basesyntax;
 
-class Pair {
+public class Pair<T1, T2> {
+    private T1 first;
+    private T2 second;
 
+    public Pair(T1 first, T2 second) {
```

Comment:
please check again the task description
does it give any requirements for constructor?

Reply:
done